apps/src/blockTooltips/DropletTooltipManager.js

```diff
@@ -0,0 +1,92 @@
+var DropletFunctionTooltip = require('./DropletFunctionTooltip');
+
+/**
+ * @fileoverview Manages a store of known blocks and tooltips
+ */
+
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+'use strict';
+
+/**
+ * Store for finding tooltips for blocks
+ * @constructor
+ */
+var DropletTooltipManager = module.exports = function () {
+  /**
+   * Map of block types to tooltip objects
+   * @type {Object.<String, DropletFunctionTooltip>}
+   */
+  this.blockTypeToTooltip = {};
+};
+
+var repositionLastTooltip = function () {
```

Comment:
Not sure if our style guide speaks on this, but my style is to use function foo() { } unless I'm already inside a function